src/layers/core/hexagon-layer/hexagon-layer.js

```diff
@@ -86,24 +83,18 @@ export default class HexagonLayer extends Layer {
     }
   }
 
-  getPickingInfo(opts) {
-    const info = super.getPickingInfo(opts);
-    const pickedCell = this.state.pickedCell;
+  _onPickSubLayer(handler, info) {
+    const pickedCell = info.picked && info.index > -1 ?
+      this.state.hexagons[info.index] : null;
```

Comment:
While I can see why this is done, it does look a bit strange: the app doesn't know anything about the aggregated cells... Yet we are suddenly exposing them to the user in the picking callback.

But I guess it doesn't make sense to send something from `props.data` back to user...